flask_resty/testing.py

```diff
@@ -1,12 +1,36 @@
+from datetime import datetime, timedelta
+import re
+
+from dateutil import parser
```

Comment:
https://github.com/toddsifleet/equals#dictionaries seems very similar to a more explicit version of what we have, no?